### PR TITLE
(docs): fix broken links

### LINKED ISSUE #359: Fix Broken links
links to eosio documentation are broken 

https://guide.eoscostarica.io/docs/eos-learn/accounts-and-permissions/

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #359 

### Steps to test
1. Run `yarn start`
2. Go to `localhost:3000`
3. Go to Developer Guidelines > Learn EOSIO > Accounts and Permissions

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/accounts-and-permissions.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ An **account** on EOSIO is a legible name for humans that get stored on the bloc
 
 The account name must comply with certain requirements: characters from `a` to `z` in lower case, numbers from `1` to `5` and have 12 characters length. E. g. `eosioaccount`.
 
-Read more on [EOSIO accounts](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions#2-accounts).
+Read more on [EOSIO accounts](https://eos-account-creator.com/choose/).
```
Comment (xavier506) [inline docs/eos-learn/accounts-and-permissions.md:13]: this seems like an EOS mainnet tool , is there a more generic EOSIO resource instead?

Comment (hanakoori01) [reply docs/eos-learn/accounts-and-permissions.md:13]: I'll check that and update it!

Comment (hanakoori01) [reply docs/eos-learn/accounts-and-permissions.md:13]: What do you think about this link instead? [new link](https://developers.eos.io/manuals/eosjs/v21.0/how-to-guides/how-to-create-an-account)

Comment (xavier506) [reply docs/eos-learn/accounts-and-permissions.md:13]: the link looks good , maybe we can change the name of the link text from `Read more on [EOSIO accounts]` to `Read more on [EOSIO account creation]`

File: docs/eos-learn/accounts-and-permissions.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Any given account can define a mapping between any of its named permissions and
 
 In other words each account's permission can be linked to an authority table used to determine whether a given action authorization can be satisfied
 
-To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions).
+To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/v2.0/protocol-guides/accounts_and_permissions).
```
Comment (xavier506) [inline docs/eos-learn/accounts-and-permissions.md:31]: shuldn't we use the `latest` version?

Comment (xavier506) [reply docs/eos-learn/accounts-and-permissions.md:31]: in other words the correct link to use should be : https://developers.eos.io/welcome/latest/protocol-guides/accounts_and_permissions

**Comment by hanakoori01** [comment]
Please do not merge this PR, I'm solving some problems with my repository.
@xavier506

**Pull request closed without merging**